apps/backend/src/models/Technique.ts

```diff
@@ -0,0 +1,15 @@
+export class Technique {
+  constructor(
+    public id: number,
```

Comment:
Should it be `techniqueId` as per the db definition? 

https://github.com/UserOfficeProject/user-office-core/pull/628/files#diff-9f73e29295089d202fe5fa3d11befff47df1c37abc181ce3ded2a113b85d7706R8

Reply:
We tried this and unfortunately couldn't get it working with the `SuperMaterialTable` component. It expects an `id` property in the `TechniqueFragment`. It seems that other objects using the same table are also using `id`. Do you know of a way around it or are we fine to leave it as `id`?